src/main/webapp/WEB-INF/jsp/tiles/views/experiments.jsp

```diff
@@ -1,73 +1,65 @@
+                                    src: "http://localhost:8080/gxa/resources/images/experiments-table/differential.png",
+                                    alt: "Microarray 1-colour miRNA"
+                                },
+                                PROTEOMICS_BASELINE: {
+                                    src: "http://localhost:8080/gxa/resources/images/experiments-table/baseline.png",
+                                    alt: "Proteomics baseline"
+                                },
+                                RNASEQ_MRNA_BASELINE: {
+                                    src: "http://localhost:8080/gxa/resources/images/experiments-table/baseline.png",
+                                    alt: "RNA-Seq mRNA baseline"
+                                },
+                                MICROARRAY_2COLOUR_MRNA_DIFFERENTIAL: {
+                                    src: "http://localhost:8080/gxa/resources/images/experiments-table/differential.png",
+                                    alt: "Microarray 2-colour mRNA"
+                                },
+                                MICROARRAY_1COLOUR_MRNA_DIFFERENTIAL: {
+                                    src: "http://localhost:8080/gxa/resources/images/experiments-table/differential.png",
+                                    alt: "Microarray 1-colour mRNA"
+                                },
+                                RNASEQ_MRNA_DIFFERENTIAL: {
+                                    src: "http://localhost:8080/gxa/resources/images/experiments-table/differential.png",
+                                    alt: "RNA-Seq mRNA differential"
+                                }
+                            }
+                        },
+                        {type: `sort`, title: `Loaded date`, width: 120, dataParam: `lastUpdate`},
+                        {type: `search`, title: `species`, width: 200, dataParam: `species`},
+                        {type: `search`, title: `experiment description`, width: 400, dataParam: `experimentDescription`,
+                            link: `experimentAccession`, resource: `experiments`, endpoint: `Results`},
+                        {type: `sort`, title: `Number of assays`, width: 120, dataParam: `numberOfAssays`,
+                            link: `experimentAccession`, resource: `experiments`, endpoint: `Experiment%20Design`},
+                        {type: `sort`, title: `Comparisons`, width: 100, dataParam: `numberOfContrasts`,
+                            link: `experimentAccession`, resource: `experiments`, endpoint: `Experiment%20Design`},
+                        {type: `search`, title: `experiment factors`, width: 200, dataParam: `experimentalFactors`},
+                        {type: `search`, title: `array designs`, width: 220, dataParam: `arrayDesignNames`,
+                            link: `arrayDesigns`, resource: `arrayexpress/arrays`, endpoint: ``}
```

Comment:
This makes the URL be resolved relative to Expression Atlas, and you get the incorrect URL `/gxa/arrayexpress/arrays/...`. Make it absolute with `https://www.ebi.ac.uk/arrayexpress/`.

Also, shouldn’t the fields here be better like this?
```javascript
resource: `https://www.ebi.ac.uk/arrayexpress`, endpoint: `arrays`
```